packages/@sfx/search-plugin/src/search-plugin.ts

```diff
@@ -0,0 +1,59 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { BridgeConfig, BrowserBridge, Query } from 'groupby-api';
+
+/**
+ * This plugin exposes an instance of the GroupBy search client
+ */
+export default class SearchPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'search',
+      depends: [],
+    };
+  }
+
+  /**
+   * The values that the Search Data Source plugin expose to the Core entity.
+   */
+  browserBridge: SearchPluginExposedValue;
+
+  /**
+   * Instantiates an instance of the search browserBridge plugin
+   *
+   * @param options The options to instantiate the search data source browser bridge client with.
+   */
+   constructor(options: SearchPluginOptions) {
+     const customerId = options.customerId;
+     const https = !!options.https;
```

Comment:
This is still incorrect if we want the default to be `true`. Consider the case where `options.https` is `undefined`, meaning that the user didn't supply it (because it's optional). `!!options.https` → `!!undefined` → `!true` → `false`.

I would instead destructure the options, like so:

```js
const {
  customerId,
  https = true
  ...browserBridgeOptions
} = options;
```

That way, `https` keeps its value if it has one, and becomes `true` if it's `undefined`. We also get `browserBridgeOptions`, which we'll use later.